daccs_client/node.py

```diff
@@ -29,15 +32,32 @@ def __init__(self, nodename: str, jsondata: dict[str]) -> None:
             self._services.append(s.name)
 
     def is_online(self) -> bool:
-        return self._status == "online"
+        try:
```

Comment:
why not just:

```python
return requests.get(self.url).ok
```

Reply:
Didn't know about that :D

Reply:
I just realized that you'd still need to catch the `ConnectionError` though I think? In case the site is completely inaccessible

Reply:
I think its also raised if the whole server is down, like if it can't even connect to that server at port 80 or 443. I'd say let's catch it just in case

Reply:
Maybe let's leave this for now, if I have to catch an exception anyway, then there is no point in changing this.